lib/pub_sub/subscriber.rb

```diff
@@ -66,5 +66,19 @@ def queue_arn(region)
     def queue_url(region)
       sqs(region).create_queue(queue_name: PubSub.service_identifier).queue_url
     end
+
+    # Collects all :topics or :subscriptions for a given SNS client
+    def self.collect_all(client, collection_type)
```

Comment:
As you rightfully pointed out earlier, I use this method out of pubsub. There, I work with a specific client (per region). Hence the discrepancy.